src/05-derivatives/03-structural-derivatives.md

```diff
@@ -0,0 +1,136 @@
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_keywords>[_hemi-{L|R}][_space-<space>]_<suffix>.{shape.gii|dscalar.nii}
+```
+
+The REQUIRED extension for scalar GiFTI files is `.shape.gii`. The `hemi` key is
+required for GiFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_curv.shape.gii
+            sub-001_hemi-R_curv.shape.gii
+```
+
+The REQUIRED extension for scalar CIFTI files is `.dscalar.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_curv.dscalar.nii
+```
+
+The file `<suffix>` should concisely describe the parameter that is represented
+in the overlay, and while the suffix can be individually customized, the
+following values should be reserved for their common use-cases:
+
+| Suffix       | Description                                           |
+| ------------ | ----------------------------------------------------- |
+| `curv`       | Cortical surface curvature indices                    |
+| `thickness`  | Cortical thickness                                    |
+| `area`       | Discretized surface area across regions               |
+| `dist`       | Distance from a point                                 |
+| `defects`    | Marked regions with surface defects                   |
+| `sulc`       | Sulcal depth                                          |
+| `myelinmap`  | Myelin map calculated from T1w to T2 ratio            |
+| `distortion` | Distortion map calculated from a surface registration |
+
+## Morphometrics
+
+Structural statistics produced by segmentation routines should be stored within
+tsv files, which could contain common parameters specified in the table below.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_keywords>[_desc-<label>]_morph.tsv
+
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_desc-volumetric_morph.tsv
+```
+
+| Column name | Description                                     |
+| ----------- | ----------------------------------------------- |
+| index       | RECOMMENDED. Label integer index                |
+| name        | RECOMMENDED. Structure name                     |
+| centroid    | OPTIONAL. Center coordinate of structure        |
+| volume      | OPTIONAL. Volume of structure                   |
+| intensity   | OPTIONAL. Intensity of voxels within structure  |
+| thickness   | OPTIONAL. Thickness of cortical structure       |
+| area        | OPTIONAL. Surface area of cortical structure    |
+| curv        | OPTIONAL. Curvature index of cortical structure |
+
+Some parameters might require unit specification or have multiple associated
+statistics (such as avg, std, min, max, range). The suggested syntax for such
+columns is `<parameter>[-<stat>][-<units>]`. An example volumetric stats file
+might look something like this:

```

Comment:
where would the atlas used to generate this be defined? say `aparc` vs `a2009s`

Reply:
This seems like it ought to be JSON sidecar. I'm not sure if it fits in any of the metadata fields we've previously described...

Reply:
Good point, but I think we could add this later if no concrete proposal materializes soon.